routes/project/billing.rb

```diff
@@ -40,10 +42,16 @@ class Clover
                 line2: nil
               },
               metadata: {
-                tax_id: r.params["tax_id"].gsub(/[^a-zA-Z0-9]/, ""),
+                tax_id: new_tax_id,
                 company_name: r.params["company_name"]
               }
             })
+            if new_tax_id != current_tax_id
+              DB.transaction do
+                billing_info.update(valid_vat: nil)
+                Strand.create(prog: "ValidateVat", label: "start", stack: [{subject_id: billing_info.id}])
```

Comment:
If `billing_info.country.in_eu_vat?` doesn't require a costly lookup, and only `billing_info.validate_vat` does, I recommend we only create a strand in the case where we need to do the `validate_vat` lookup.